libraries/AP_Motors/AP_MotorsMatrix.cpp

```diff
@@ -414,7 +414,7 @@ void AP_MotorsMatrix::output_armed_stabilizing()
 void AP_MotorsMatrix::check_for_failed_motor(float throttle_thrust_best_plus_adj)
 {
     // record filtered and scaled thrust output for motor loss monitoring purposes
-    float alpha = 1.0f / (1.0f + _loop_rate * 0.5f);
+    float alpha = _dt / (_dt + 0.5f);
```

Comment:
This change results in about double the alpha value when loop_rate = 400.  Is this an intentional change?

Reply:
lr =  400
>> 1/(1+lr*0.5)
ans =  0.0050
>> dt = 1/lr
dt =  0.0025
>> dt / (dt + 0.5)
ans =  0.0050

I think this one is right.

Reply:
I'm sorry, you're right.  I had a typo when I did my own check.